plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -535,7 +540,7 @@ function( $feature_id ) use ( $features ) {
 				return $features[ $feature_id ]['is_experimental'];
 			}
 		);
-		$mature_feature_ids       = array_diff( $feature_ids, $experimental_feature_ids );
+		$mature_feature_ids       = array_diff( $feature_ids, $experimental_feature_ids, array( 'cart_checkout_blocks' ) );
```

Comment:
Isn't `cart_checkout_blocks` a key of `$features` already, meaning `$feature_ids` would include it? Is there a reason you had to specifically add it here?

Reply:
Hi @opr! Thank you for the review! 🙌 

> Isn't cart_checkout_blocks a key of $features already, meaning $feature_ids would include it? Is there a reason you had to specifically add it here?

In this line, we are removing the `cart_checkout_blocks` to exclude it from the `mature_feature_ids` list. We don't want the add a `setting UI` for the Cart & Checkout feature. We will get the following result without this line:

<img width="1123" alt="image" src="https://user-images.githubusercontent.com/14235870/212893943-09d91df5-5741-4c85-9ad8-089e94c12558.png">

Reply:
Great, thanks for explaining!